### ISSUE TITLE
What does this mean

### ISSUE THREAD

**Issue by peanutwu** [ItIckeYd/VodRecovery#28]
I tried to open the RecoverVod.py and i wanna get the m3u8 link *(The VOD m3u8 link) 
I entered the streamer name and vod id and then  show 

Enter VOD timestamp (YYYY-MM-DD HH:MM:SS):

what should i enter it .  Shouldn't it pop out a m3u8 link of the vod ? why should i enter the timestamp. 

and how can i solve it . thank you

**Comment by ItIckeYd** [comment]
@peanutwu You need to enter the timestamp its a requirement for the script to run. timestamp is the start time of the vod which can be found on twitchtracker, sullygnome or streamscharts. When entering the timestamp input the seconds value as 00.

Note: The timestamp is required in local time... therefore whatever the websites I listed above say you will need to add or minus the UTC offset of your local timezone.

Example: 
UTC start time  -  9:00:00

Korea is UTC+9
Korea start time  - 18:00:00 --This is the one you would use.

**Comment by peanutwu** [comment]
SAME WITH https://github.com/TwitchRecover/TwitchRecover/issues/235

I PUT IT IN and nothing happened  the windows disappear.

**Comment by peanutwu** [comment]
command . also i downloaded the TwtichRecover and it worked.   now it can only recover some of the vods. 
example.  same steamer -   https://www.twitch.tv/woohankyung
i opened twitchRecover and 3-1 and put  https://www.twitch.tv/videos/1439211562   i can  get the m3u8 link
but when i put in the latest vod like    https://www.twitch.tv/videos/1310367080     (stream 12h ago) 
the TwtichRecover will disappear.   i found this about 1-2 weeks ago  .

**Comment by peanutwu** [comment]
oh sorry for my poor english .

**Comment by ItIckeYd** [comment]
@peanutwu Your not doing something right.. Please give an example of input.

**Comment by peanutwu** [comment]
MY LOCAL TIME IS UTC-4  
streamscharts show these
Start time:
26 May 2022, 08:12 (UTC)

so i put 2022-05-26 04:12:00  right ?

**Comment by peanutwu** [comment]
C:\Users\Administrator>python C:\Users\Administrator\Downloads\VodRecovery-main\VodRecovery-main\RecoverVod.py
Enter streamer name: runner0608
Enter vod id: 1309376781
Enter VOD timestamp (YYYY-MM-DD HH:MM:SS): 2022-05-26 04:12:00
No vods found using current domain list.

**Comment by ItIckeYd** [comment]
@peanutwu https://streamscharts.com/channels/runner0608/streams/1309376781 doesn't exist... first of all.

**Comment by peanutwu** [comment]
https://streamscharts.com/channels/runner0608/streams/39356780103  isnt this  - - ? omg

**Comment by ItIckeYd** [comment]
@peanutwu Your timezone is incorrect. Works when I do it. Figure out your actual timezone then run it.

**Comment by peanutwu** [comment]
SORRY I WILL DO IT AGAIN

**Comment by ItIckeYd** [comment]
@peanutwu why cap everything.. I wrote the script I know it works lol.

**Comment by peanutwu** [comment]
https://www.twitch.tv/videos/1309376781   
is 1309376781 the vod id ?

**Comment by ItIckeYd** [comment]
@peanutwu no.. its the one from twitchtracker or streamscharts

**Comment by peanutwu** [comment]
why i check in streamscharts    
https://streamscharts.com/channels/runner0608/streams/39356780103

**Comment by peanutwu** [comment]
they re different

**Comment by ItIckeYd** [comment]
@peanutwu yes, that is how its supposed to be.

**Comment by peanutwu** [comment]
so i need to put in  39356780103 but 130937681      39356780103 is right...

**Comment by peanutwu** [comment]
let me try again..

**Comment by ItIckeYd** [comment]
@peanutwu no idea what that means.

**Comment by peanutwu** [comment]
can you give me what you  put  in

**Comment by peanutwu** [comment]
Enter streamer name: runner0608
Enter vod id: 1309376781
Enter VOD timestamp (YYYY-MM-DD HH:MM:SS): 2022-05-26 04:12:00
No vods found using current domain list.

**Comment by peanutwu** [comment]
like these  👆

**Comment by ItIckeYd** [comment]
@peanutwu Your timezone is incorrect Figure out your timezone lol.

**Comment by peanutwu** [comment]
i put the right timezone lol  i found it UTC-5

**Comment by ItIckeYd** [comment]
@peanutwu 8-5 is 3 not 4

**Comment by peanutwu** [comment]
Enter streamer name: runner0608
Enter vod id: 39356780103
Enter VOD timestamp (YYYY-MM-DD HH:MM:SS): 2022-05-26 03:12:00
No vods found using current domain list.   


Still

**Comment by peanutwu** [comment]
cry..

**Comment by ItIckeYd** [comment]
@peanutwu where do you live? give me a state or something ill figure out what to put in.

**Comment by peanutwu** [comment]
my time is May 27    2:20 PM now

**Comment by peanutwu** [comment]
i need example and i hope i will know how it works

**Comment by peanutwu** [comment]
please use this link for example   https://www.twitch.tv/videos/1309376781

**Comment by ItIckeYd** [comment]
@peanutwu
Enter streamer name: runner0608
Enter vod id: 39356780103
Enter VOD timestamp (YYYY-MM-DD HH:MM:SS): 2022-05-26 04:12:00


that will work.

**Comment by peanutwu** [comment]
yes it works

**Comment by peanutwu** [comment]
let me think for awhile

**Comment by ItIckeYd** [comment]
@peanutwu get it? use the proper website with the proper timezone and it works.

**Comment by peanutwu** [comment]
i used to use TwitchRecover to get the m3u8 link .   but now it only can get the old vods' link

**Comment by ItIckeYd** [comment]
#67
